src/napari_micromanager/_mda_handler.py

```diff
@@ -163,11 +194,15 @@ def _on_mda_frame(self, image: np.ndarray, event: MDAEvent) -> None:
             layer: Image = self.viewer.layers[layer_name]
             if not layer.visible:
                 layer.visible = True
-            # layer.reset_contrast_limits()
+            layer.reset_contrast_limits()
```

Comment:
accidental uncomment
```suggestion
            # layer.reset_contrast_limits()
```